daphne_server/src/router/aggregator.rs

```diff
@@ -107,4 +148,81 @@ mod test {
 
         assert_eq!(status, StatusCode::OK);
     }
+
+    // Check that a signature produced by Daphne-Worker will be verified properly by the Clients.
+    #[test]
+    fn rondtrip_sign_hpke_config() {
+        fn os<S: AsRef<OsStr> + ?Sized>(s: &S) -> &OsStr {
+            OsStr::new(s)
+        }
+
+        let mut signing_key_file = tempfile::NamedTempFile::new().unwrap();
+        let status = Command::new("openssl")
+            .args([
+                os("ecparam"),
+                os("-name"),
+                os("prime256v1"),
+                os("-genkey"),
+                os("-noout"),
+                os("-out"),
+                os(signing_key_file.path()),
```

Comment:
Instead of writing to a temporary file, you could write to stdout and capture it here.

Reply:
No point in doing that since I need the key file for the next command anyway